app/core/src/main/java/com/fsck/k9/message/extractors/PreviewTextExtractor.kt

```diff
@@ -99,6 +98,8 @@ internal class PreviewTextExtractor {
     private val EmailSection.quoteHeaderIndex: Int
         get() {
             var quoteHeaderIndex = lastIndex
+            if (quoteHeaderIndex == -1) return -1
```

Comment:
An empty body (handled in `extractUnquotedText()`) *should* be the only way to create an empty `EmailSection`. This check is to make sure we don't crash if that assumption doesn't hold (in the future).